style.css

```diff
@@ -0,0 +1,131 @@
+/* Edit */
+
+h1~p {
+  color: black;
+}
+
+h1 {
+  color: black;
+}
+
+/* Varför fungerar inte detta? */
```

Comment:
The reason it's not working is because ~ is a sibling relationship, whereas you're trying to create a parent relationship. Earlier you used `h1~p` because `h1` and `p` are next to each other on the same level, but here you have `h1` nested inside of `section`, so you have to use a nesting selector like `>`.

Reply:
It would be `section > h1`